crates/ruff_python_ast/src/nodes.rs

```diff
@@ -4061,7 +4061,7 @@ impl Ranged for Identifier {
     }
 }
 
-#[derive(Clone, Debug, PartialEq)]
+#[derive(Clone, Copy, Debug, PartialEq)]
```

Comment:
this change seems worth doing on its own -- could consider splitting it off into a separate refactor. (But it's also pretty small, so I don't mind it being part of this PR.)